kml/serdes_gpx.cpp

```diff
@@ -140,6 +142,39 @@ void GpxParser::ParseColor(std::string const & value)
   m_color = kml::ToRGBA(colorBytes[0], colorBytes[1], colorBytes[2], (char)255);
 }
 
+// https://osmand.net/docs/technical/osmand-file-formats/osmand-gpx/ - "#AARRGGBB" or "#RRGGBB"
+void GpxParser::ParseOsmandColor(std::string const & value) {
+  if (value.empty())
+  {
+    LOG(LWARNING, ("Empty color value"));
+    return;
+  }
+  auto const colorBytes = FromHex(value.substr(1, value.size() - 1));
+  uint32_t color = kInvalidColor;
```

Comment:
nit: According to the code below, this initialization is unnecessary.

Reply:
Fixed